content/site.json

```diff
@@ -1,16 +1,12 @@
 {
   "site": {
-    "siteTitle": "MVG Portal Demonstrator",
-    "siteTagline": "A platform to find, publish and consume Data Services in the Gaia-X Test Network.",
+    "siteTitle": "EuProGigant Portal",
+    "siteTagline": "The lighthouse project for Gaia-X in manufacturing",
```

Comment:
suggestion(non-blocking): This sounds a little strange to my ears. Maybe something like `The Gaia-X lighthouse project for manufacturing` fits better here.